src/sentry/models/eventerror.py

```diff
@@ -94,6 +95,7 @@ class EventError(object):
         NATIVE_MISSING_SYMBOL: u'Could not resolve one or more frames in debug information file.',
         NATIVE_SIMULATOR_FRAME: u'Encountered an unprocessable simulator frame.',
         NATIVE_UNKNOWN_IMAGE: u'A binary image is referenced that is unknown.',
+        NATIVE_FAILED: u'Failed to process.',
         NATIVE_SYMBOLICATOR_FAILED: u'Failed to call Sentry-internal service.',
```

Comment:
Let's use `NATIVE_SYMBOLICATOR_FAILED` for all cases and change this to:

```suggestion
        NATIVE_SYMBOLICATOR_FAILED: u'Failed to process native stacktraces.',
```